freqtrade/exchange/exchange.py

```diff
@@ -1834,7 +1853,7 @@ def _set_leverage(
             return
 
         try:
-            self._api.set_leverage(symbol=pair, leverage=leverage)
+            self._api.set_leverage(symbol=pair, leverage=round(leverage))
```

Comment:
are you sure set_leverage must be a full number? (to be precise - are you sure that's the case for all exchanges?).